packages/fxa-settings/src/lib/glean/index.test.ts

```diff
@@ -0,0 +1,523 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Glean from '@mozilla/glean/web';
```

Comment:
How difficult is it to pull this into fxa-shared and be used in both settings and content-server?

Reply:
I've been working from the assumption content-server will be go away soon-ish so I didn't put _that much_ effort into moving everything possible into fxa-shared.  As for this specifically, because how and where the Glean metrics module get its metrics values, we'd need to add another level of indirection so the code in fxa-shared presents an interface to content-server and fxa-settings to provide some metrics value getters.  Do-able, not sure if worth it.  Let me know if I'm overthinking this though. 😀

Reply:
I don't know when the content-server will go away, but doubt it is soon-ish. I can't image we will be implementing any new features for content-server though, so this might be alright.

Reply:
> I don't know when the content-server will go away, but doubt it is soon-ish.

Ok, I thought we are in-progress with the remaining routes.  If that's not the case and a Glean bug pops up then I'll see about moving the lib into fxa-shared and fix it in one place at that point.